smart-connector-rest-server/src/main/java/eu/knowledge/engine/rest/api/impl/ProactiveApiServiceImpl.java

```diff
@@ -57,7 +177,7 @@ public void scAskPost(
 			@Parameter(description = "The keys bindings are allowed to be incomplete, but they must correspond to the binding keys that were defined in the knowledge interaction.", required = true) @NotNull @Valid JsonNode recipientAndBindingSet,
 			@Suspended final AsyncResponse asyncResponse, @Context SecurityContext securityContext) {
 
-		LOG.debug("scAskPost called for KB {} and KI {} - {}", knowledgeBaseId, knowledgeInteractionId,
+		LOG.info("scAskPost called for KB {} and KI {} - {}", knowledgeBaseId, knowledgeInteractionId,
```

Comment:
Before merging, these should be made `debug` again, otherwise the logs output get too cluttered.

Reply:
Done!